src/core/evaluator.js

```diff
@@ -2572,6 +2593,8 @@ class PartialEvaluator {
 
       const glyphs = font.charsToGlyphs(chars);
       const scale = textState.fontMatrix[0] * textState.fontSize;
+      const whitespaces = glyphs.map(glyph => isWhitespace(glyph.unicode));
```

Comment:
*One question:* Rather than looping through the glyphs here, would it possibly make sense (and be more efficient) to add a new property (e.g. `isWhitespace` or similar) to the `Glyph`-instances and utilize the new helper function there instead to determine this for each glyph as part of the existing parsing in https://github.com/mozilla/pdf.js/blob/master/src/core/fonts.js#L3153?

That would obviously require extending the `Glyph` class, but given the existence of both a charCache *and* a glyphCache in the font code that should actually help reduce the overall amount of parsing required.

Reply:
Good idea.